README.aipl

```diff
@@ -0,0 +1,255 @@
+!!def aipl-ops
+ !python-input>>op
+    aipl.operators.values()
+ !format
+    - {op.__name__} (in={op.rankin} out={op.rankout}): {op.__doc__}
+ !join sep=\n
+
+!!def read-summary
+  !read chains/summarize.aipl
+
+
+!!read-summary>summarize_aipl
```

Comment:
With these `!!`, `summarize_aipl` is added to the global context, with the contents of `!read chains/summarize.sipl`. Then, you can reference it later in the pipeline via `summarize_aipl`.